superset/config.py

```diff
@@ -190,6 +190,7 @@
 DEFAULT_MODULE_DS_MAP = OrderedDict([
     ('superset.connectors.sqla.models', ['SqlaTable']),
     ('superset.connectors.druid.models', ['DruidDatasource']),
+    ('superset.connectors.elastic.models', ['ElasticDatasource']),
```

Comment:
Maybe is good don't short the name and call it `.elasticsearch.` and `ElasticseachDatasource` to avoid confusion with other future datasources.
I don't know at the moment, but maybe in the future people wants integration with other products, for example, _Elastic Query_ from Azure.